Modules/_ctypes/stgdict.c

```diff
@@ -745,6 +754,8 @@ PyCStructUnionType_update_stgdict(PyObject *type, PyObject *fields, int isStruct
          *     struct { uint_32 e1; uint_32 e2; ... uint_32 e_4; } f6;
          * }
          *
+         * The same principle applies for a struct 32 bytes in size.
```

Comment:
On Arm, or everywhere?

Reply:
The 32 bytes happen only on Arm platforms. Again I can make the comment more explicit.

Reply:
This line I would prefer was more explicit.

Reply:
This is done now.